evaluation/promptfooconfig.yaml

```diff
@@ -0,0 +1,15 @@
+# This configuration compares LLM output of 2 prompts x 2 GPT models across 3 test cases.
+# Learn more: https://promptfoo.dev/docs/configuration/guide
+description: 'Health Prompt Eval'
+
+prompts:
+  - file://prompt_no_context.txt
```

Comment:
Are these files.. somewhere?

Reply:
They're generated by `comparing_prompt_with_and_without_context.py`. You'd change those for whatever test you were doing. It's possible you can make multiple config files too, I'm not 100% sure